ui/lib/apps/Statement/translations/en.yaml

```diff
@@ -31,3 +31,23 @@ statement:
     last_seen: Last Seen
   plan:
     plans: Plans
+  tooltip:
+    refresh: Refresh
+  setting:
+    title: Settings
+    disabled_desc_title: This Feature Is Not Open
+    disabled_desc_line_1: SQL statements feature is not open, we can't view history records.
+    disabled_desc_line_2: You can change the setting to open it and wait for the new records.
+    open_setting: Open Setting
+    close_statement: Close SQL statements
+    close_statement_warning: Are you sure to close this feature? All history records will be cleared after closing it!
+    switch: Open Statements
+    refresh_interval: Data Collect Interval
+    keep_duration: Data Keep Duration
+  actions:
+    save: Save
+    close: Close
+    cancel: Cancel
+  time:
```

Comment:
I have a lazy idea.. We can use `getValueFormat` / prettyMs to display time. We can avoid writing min / day translations again and again. Actually there is no need to translate the time unit to Chinese since everyone knows it.

Reply:
Ok, let me have a try.

Reply:
`getValueFormat` converts "17  day" to "2 week" or "2.4 week", I don't think it is ok for me, and I think it is acceptable to fix the unit to "day" in this situation. I think it is fine to fix the unit to "min" and "day" in this situation.